src/components/Menu/ReactSelectStyles/dropdown-indicator-styles.ts

```diff
@@ -12,3 +12,11 @@ export const dropdownIndicatorStyles: StylesConfigFunction<
     color: 'var(--color-gui-hover)',
   },
 });
+
+export const dropdownIndicatorDisabledStyles: StylesConfigFunction<
+  DropdownIndicatorProps<ReactSelectOption, false, ReactSelectOptGroup>
+> = (provided) => ({
+  ...provided,
+  padding: '4px',
+  color: 'var(--color-mid-grey)',
```

Comment:
Shame this doesn't tie in to our Tailwind design system but given that it's `react-select`'s way of doing things, it is what it is